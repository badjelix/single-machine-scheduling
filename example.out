mapa:
1 : ('t', 1)
2 : ('t', 2)
3 : ('t', 3)
4 : ('t', 4)
5 : ('k', 1, 1, 0)
6 : ('k', 1, 1, 1)
7 : ('k', 1, 1, 2)
8 : ('k', 1, 1, 3)
9 : ('k', 1, 1, 4)
10 : ('k', 1, 1, 5)
11 : ('k', 1, 2, 0)
12 : ('k', 1, 2, 1)
13 : ('k', 1, 2, 2)
14 : ('k', 1, 2, 3)
15 : ('k', 1, 2, 4)
16 : ('k', 1, 2, 5)
17 : ('k', 1, 3, 0)
18 : ('k', 1, 3, 1)
19 : ('k', 1, 3, 2)
20 : ('k', 1, 3, 3)
21 : ('k', 1, 3, 4)
22 : ('k', 1, 3, 5)
23 : ('k', 2, 1, 2)
24 : ('k', 2, 1, 3)
25 : ('k', 2, 1, 4)
26 : ('k', 2, 2, 2)
27 : ('k', 2, 2, 3)
28 : ('k', 2, 2, 4)
29 : ('k', 3, 1, 3)
30 : ('k', 3, 1, 4)
31 : ('k', 3, 1, 5)
32 : ('k', 3, 1, 6)
33 : ('k', 3, 1, 7)
34 : ('k', 3, 1, 8)
35 : ('k', 3, 2, 3)
36 : ('k', 3, 2, 4)
37 : ('k', 3, 2, 5)
38 : ('k', 3, 2, 6)
39 : ('k', 3, 2, 7)
40 : ('k', 3, 2, 8)
41 : ('k', 4, 1, 5)
42 : ('k', 4, 1, 6)
43 : ('k', 4, 1, 7)
44 : ('k', 4, 2, 5)
45 : ('k', 4, 2, 6)
46 : ('k', 4, 2, 7)
c
c Open-WBO:	 a Modular MaxSAT Solver -- based on Glucose4.1 (core version)
c Version:	 September 2018 -- Release: 2.1
c Authors:	 Ruben Martins, Vasco Manquinho, Ines Lynce
c Contributors:	 Miguel Neves, Saurabh Joshi, Norbert Manthey, Mikolas Janota
c Contact:	 open-wbo@sat.inesc-id.pt -- http://sat.inesc-id.pt/open-wbo/
c
c WARNING: for repeatability, setting FPU to use double precision
c Warning: no filename.
c |                                                                                                       |
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Problem Format:             MaxSAT                                                                   |
c |  Problem Type:           Unweighted                                                                   |
c |  Number of variables:            46                                                                   |
c |  Number of hard clauses:        227                                                                   |
c |  Number of soft clauses:          4                                                                   |
c |  Number of cardinality:           0                                                                   |
c |  Number of PB :                   0                                                                   |
c |  Parse time:                   0.00 s                                                                 |
c |                                                                                                       |
c ==========================================[ Solver Settings ]============================================
c |                                                                                                       |
c |  Algorithm:                    MSU3                                                                   |
c |  Cardinality Encoding:    Totalizer                                                                   |
o 2
o 1
s OPTIMUM FOUND
v 1 -2 3 4 5 6 -7 -8 -9 -10 -11 -12 13 -14 -15 -16 -17 -18 -19 20 -21 -22 -23 -24 -25 -26 -27 -28 -29 30 -31 -32 -33 -34 -35 -36 -37 -38 -39 40 41 -42 -43 -44 45 46 

[1, -2, 3, 4, 5, 6, -7, -8, -9, -10, -11, -12, 13, -14, -15, -16, -17, -18, -19, 20, -21, -22, -23, -24, -25, -26, -27, -28, -29, 30, -31, -32, -33, -34, -35, -36, -37, -38, -39, 40, 41, -42, -43, -44, 45, 46]
('t', 1)
('t', 3)
('t', 4)
('k', 1, 1, 0)
('k', 1, 1, 1)
('k', 1, 2, 2)
('k', 1, 3, 3)
('k', 3, 1, 4)
('k', 3, 2, 8)
('k', 4, 1, 5)
('k', 4, 2, 6)
('k', 4, 2, 7)
3
1 0 2 3
3 4 8
4 5 6

