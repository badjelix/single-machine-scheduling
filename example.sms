4
0 4 6 3 2 1 1
2 3 5 2 2 1
3 2 9 2 1 1
5 3 8 2 1 2
0
0
1 1
0
