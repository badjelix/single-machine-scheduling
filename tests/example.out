3
1 0 2 3
3 4 8
4 5 6

